src/utils/cd.js

```diff
@@ -217,6 +217,16 @@ export function validateh(rules, proof) {
   };
 }
 
+// A wrapper around infer that first converts premises and rules to the canonical representation
+// as defined by `canon()` in `claimgraph.js`. This wrapper deserializes the returned values before
+// passing them back to the caller.
+export function inferh(premises, rules) {
```

Comment:
not sure on the naming of this method, `inferh` isnt verify descriptive. what does the `h` stand for, if anything?

Reply:
I can add a comment stating that it's short for infer-high because it's a higher level wrapper around `infer`.
The same is true for `proveh` and `validateh`.

Reply:
i think we can just leave as is, dont think its something user would care about anyway. h for high level wrapper can do

Reply:
Too late. :)
c12a16b